continuous_integration/travis/runtests.sh

```diff
@@ -1,38 +1,40 @@
 #!/usr/bin/env sh
-# This script is meant to be called by the "script" step defined in
-# .travis.yml. See http://docs.travis-ci.com/ for more details.
 
 # License: 3-clause BSD
 
 set -e
-which $PYTHON
-$PYTHON --version
-echo $TOXENV
+which python
+python --version
+echo ${TOX_ENV}
 
 # Make sure that we have the python docker image cached locally to avoid
 # a timeout in a test that needs it.
 
-if [ "$(which docker)" != "" ]; then
+if [ "$(which docker)" != "" ] && [ "$(uname)" = "Linux" ]; then
```

Comment:
I believe this should be:

```suggestion
if [ "$(which docker)" != "" ] && [ "$(uname)" == "Linux" ]; then
```

https://tecadmin.net/tutorial/bash/examples/check-if-two-strings-are-equal/

Reply:
`=` and `==` act the same way in `bash` according to [this SO answer](https://stackoverflow.com/questions/669452/is-double-square-brackets-preferable-over-single-square-brackets-in-ba).  I agree we might as well use `[[ ... ]]` and `==` which are more canonical constructs for `bash` script it seems.

Reply:
Indeed with single `[` the single `=` condition seems fine. Do as you wish.